website/api/recursion.md

```diff
@@ -0,0 +1,148 @@
+# Recursive Proving
+
+RISC Zero's zkVM uses recursive proving in order to achieve unbounded computation size, constant proof size, proof aggregation, and proof composition.
+
+:::tip
+
+[Prover::prove_with_opts][prove_with_opts] allows users to choose between [composite, succinct or groth16 receipts].
+
+:::
+
+The rest of this page describes low-level details that are not necessary for users.
+
+## Recursive Proving Process
+
+The end-to-end process for proof generation is shown in the following diagram.
+
+```mermaid
+flowchart TB
+
+execute(".execute()")
+--> Session
+
+subgraph Session["<a href='https://docs.rs/risc0-zkvm/latest/risc0_zkvm/struct.Session.html'>Session</a>"]
+direction TB
+s1("<a href='https://docs.rs/risc0-zkvm/latest/risc0_zkvm/struct.Segment.html'>Segment</a>")
```

Comment:
do you want all these links to open in the same window? I typically add target to open in a new tab:


```suggestion
s1("<a target="_blank" href='https://docs.rs/risc0-zkvm/latest/risc0_zkvm/struct.Segment.html'>Segment</a>")
```

If you wanna, this should be a quick find and replace - I am looking too to see if we can declare labels instead of the full URL every time :crossed_fingers:

Reply:
good call